### PR TITLE
Restructure

### PR DESCRIPTION
**Pull request by rashley-iqt**
- refactored file structure for readability and maintainability
- added separate flake8 config
- remediated most flake 8 flagged issues

### CODE REVIEW

File: daisybell/__main__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from tabulate import tabulate
 import pandas as pd
 
-from daisybell import scan
+from daisybell.daisybell import scan
```
Comment (jbenjoseph) [inline daisybell/__main__.py:10]: Probably should be at least imported in __init__.py so it's 

 `from daisybell import scan`

File: daisybell/scanners/ner_language_bias.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+import pysbd
+from tqdm.auto import tqdm
+from transformers import Pipeline
+
+from daisybell.daisybell import scanner
+from daisybell.helpers.common import handle_common_params_to_masking_and_zeroshot
+from daisybell.helpers.dataset import emit_books, replace_entities
+
+
+@scanner(
```
Comment (jbenjoseph) [inline daisybell/scanners/ner_language_bias.py:11]: Does the annotations work when in separate files? I had bad luck with this.

Try running daisybell from the command line eg.

`daisybell roberta-base`

Or on a GPU machine (better)

`daisybell roberta-base -d cuda`

If the command crashes or returns very fast without scanner output, it means none of the scanners registered.

File: daisybell/__main__.py (hunk 1 above)
Comment (rashley-iqt) [reply daisybell/__main__.py]: Done

File: daisybell/scanners/ner_language_bias.py (hunk 2 above)
Comment (rashley-iqt) [reply daisybell/scanners/ner_language_bias.py]: In the latest commits I switched everything over to a plugin based model. This now works with the exact usage pattern that you've described above.

File: daisybell/daisybell.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    handler = logging.handlers.WatchedFileHandler(
+        os.environ.get("LOGFILE", "./daisybell.log"))
+    formatter = logging.Formatter(logging.BASIC_FORMAT)
+    handler.setFormatter(formatter)
+    root_logger = logging.getLogger()
+    root_logger.setLevel(os.environ.get("LOGLEVEL", "INFO"))
+    root_logger.addHandler(handler)
+
+    # registered_scanners = collect_scanners(MODULES, root_logger)
```
Comment (jbenjoseph) [inline daisybell/daisybell.py:30]: commented code

File: tests/test_scanner.py [hunk 4]
```diff
@@ -1,13 +1,16 @@
+import logging
+
 from daisybell import scan
+# from daisybell.scanners import MaskingLanguageBias, NerLanguageBias, ZeroShotLanguageBias
 from transformers import pipeline
 
 
 def test_scanning_masking_human_bias():
+    # mlb = MaskingLanguageBias()
```
Comment (jbenjoseph) [inline tests/test_scanner.py:9]: commented code

File: tests/test_scanner.py [hunk 5]
```diff
@@ -1,13 +1,16 @@
+import logging
+
 from daisybell import scan
+# from daisybell.scanners import MaskingLanguageBias, NerLanguageBias, ZeroShotLanguageBias
```
Comment (jbenjoseph) [inline tests/test_scanner.py:4]: commented code

**Review by jbenjoseph — APPROVED** [review]
Looks good

**Pull request merged**